lisa/tools/rpm.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import cast
```

Comment:
This should be in base_tools folder. We can use it in operating system classes later.

Reply:
update